### ISSUE TITLE
Event Hubs client raises error and close driver after idle time

### ISSUE THREAD

**Issue by ToonUp** [Azure/azure-event-hubs-spark#340]
Using HDInsight with Spark 2.2.0 and the EH connector 2.2.0. If there are no events coming from the Event Hub, the Spark Structured Streaming application - after some hours of idle time - is closed by the following EventHubsClient's error:

```
java.util.concurrent.ExecutionException: com.microsoft.azure.eventhubs.EventHubException: 
The connection was inactive for more than the allowed 300000 milliseconds and is closed by container 'LinkTracker'. TrackingId:021051dfcdb04b4ba00c828369cded74_G10, SystemTracker:gateway5, Timestamp:6/10/2018 3:02:59 AM
        at java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:357)
        at java.util.concurrent.CompletableFuture.get(CompletableFuture.java:1895)
        at org.apache.spark.eventhubs.client.EventHubsClient.getRunTimeInfo(EventHubsClient.scala:112)
        at org.apache.spark.eventhubs.client.EventHubsClient.boundedSeqNos(EventHubsClient.scala:149)
        at org.apache.spark.sql.eventhubs.EventHubsSource$$anonfun$6.apply(EventHubsSource.scala:130)
        at org.apache.spark.sql.eventhubs.EventHubsSource$$anonfun$6.apply(EventHubsSource.scala:128)
```

**Comment by sabeegrewal** [comment]
Hey @ToonUp - this is fixed in #323. When the underlying connection is closed, we'll force it to re-establish and continue. This will be released in v2.3.2 and v2.2.1 in about a week! 

Let me know if you have any questions/concerns 👍 [gitter](https://gitter.im/azure-event-hubs-spark/Lobby) is the ideal place for more discussion!

**Comment by ytaous** [comment]
Hi, Sabee, we are using structured streaming on DataBricks. The SDK we have is 2.3.12, and it's reproducible:

19/10/26 00:21:35 WARN RequestResponseOpener: requestResponseChannel.onClose error clientId[MF_0a1146_1572031611448], session[mgmt-session], link[mgmt], endpoint[$management], error com.microsoft.azure.eventhubs.EventHubException: The connection was inactive for more than the allowed 300000 milliseconds and is closed by container 'LinkTracker'. TrackingId:5fd5964556f4458499f35c517e4cb5e6_G0, SystemTracker:gateway5, Timestamp:2019-10-26T00:21:35

Any idea?
